CHANGELOG_PENDING.md

```diff
@@ -18,5 +18,6 @@ program](https://hackerone.com/tendermint).
 ### FEATURES:
 
 ### IMPROVEMENTS:
+- Recover from application panics in `abci/server/socket_server.go` to allow socket cleanup.
```

Comment:
```suggestion

- [abci] Recover from application panics in `server/socket_server.go` to allow socket cleanup (@ruseinov)
```

Reply:
this is addressed, I didn't realise I can commit it directly from here :/